install/error-handling.sh

```diff
@@ -1,10 +1,51 @@
 echo "${_group}Setting up error handling ..."
 
+export SENTRY_DSN='https://19555c489ded4769978daae92f2346ca@self-hosted.getsentry.net/3'
+export SENTRY_ORG=self-hosted
+export SENTRY_PROJECT=installer
+
+function send_event {
+  if [[ $DOCKER_PLATFORM == "linux/amd64" ]]; then
+    local sentry_cli="docker run --rm -v \$(pwd):/work -e SENTRY_DSN=$SENTRY_DSN getsentry/sentry-cli"
+  else
+    if ! command -v sentry-cli &> /dev/null; then
+        echo "sentry-cli could not be found, please install it"
+        exit 1
+    fi
+    local sentry_cli=sentry-cli
+  fi
+  command pushd .. > /dev/null
+  $sentry_cli send-event --no-environ -f "$1" -m "$2" --logfile $log_file
+  command popd > /dev/null
+}
+
+if [[ -f .reporterrors ]]; then
+  if [[ "$(cat .reporterrors)" == "yes" ]]; then
+    export REPORT_ERRORS=1
+  else
+    export REPORT_ERRORS=0
+  fi
+else
+  echo "Would you like to opt-in to error monitoring for the Sentry installer?"
+  echo "This helps us catch and fix errors when installing Sentry."
+  echo "We may collect and retain your OS username, IP address, and installer log, for 30 days."
+  echo "Your information is solely used to improve the installer and is subject to our privacy policy[1]."
+  echo "You may change your preference at any time by deleting the '.reporterrors' file."
```

Comment:
This is a lie-ish. The `.reporterrors` file ends up under `install/`. I would expect to be in the distribution root. No?

Reply:
Yeah, this is a good point, since the installer mostly runs things in the `install` folder. I'll change it so it places it in the distribution root.